lib/reporter.js

```diff
@@ -109,6 +118,11 @@ var generateReport = function(options) {
                 element.ambiguous = 0;
                 element.time = 0;
 
+                if(element.type=="background")
```

Comment:
Can you please replace double quotes with single quotes? 
Also the if should be `===` 

```
element.type === 'background'
```